honeycombio/libhoney-js
# maint: add smoke test

## Which problem is this PR solving?

- closes #254 
- run a trivial example and fail if it fails to run

## Short description of the changes

- inspired by https://github.com/honeycombio/libhoney-py/pull/167

## How to verify

- `make smoke` locally
- to see how it might fail: https://app.circleci.com/pipelines/github/honeycombio/libhoney-js/1794/workflows/d02f7b68-ce90-46e3-99b5-222292e0a713/jobs/10895
